### PR TITLE
New status in docs.py and http docs created with it.

### PR DESCRIPTION
**Pull request by oleksiyVeretiuk**

### CODE REVIEW

File: docs.py [hunk 1]
```diff
@@ -146,13 +146,24 @@ def test_docs_tutorial_with_concierge(self):
 
         self.app.authorization = ('Basic', ('broker', ''))
 
+        # Switch to 'pending.dissolution'
+        #
+        with open('docs/source/tutorial/patch-lot-to-pending.dissolution.http', 'w') as self.app.file_obj:
+            response = self.app.patch_json('/{}?acc_token={}'.format(lot_id, owner_token),
+                                           {'data': {"status": 'pending.dissolution'}})
+            self.assertEqual(response.status, '200 OK')
```
Comment (yarsanich) [inline docs.py]: Потрібно ще перевірити, що у відповіді `response.json['data']` `status` дійсно змінився.

Comment (oleksiyVeretiuk) [reply docs.py]: https://github.com/openprocurement/openregistry.lots.basic/pull/39/commits/c46b59a75dffd8a2950928e3d561c045558803be

File: docs/source/tutorial.rst [hunk 2]
```diff
@@ -79,12 +79,13 @@ The previous lot contained only required fields. Let's try creating lot with mor
...
 
-.. XXX patching lot to dissolved
+.. XXX patching lot to pending.dissolution
 
-.. literalinclude:: tutorial/patch-lot-to-dissolved.http
+.. literalinclude:: tutorial/patch-lot-to-pending.dissolution.http
    :language: javascript
 
 And again we have `201 Created` response code, `Location` header and body with extra `id`, `lotID`, and `dateModified` properties.
+Only bot can change status to `dissolved`.
```
Comment (ivanka-jakimchuk) [inline docs/source/tutorial.rst]: For lot to be dissolved you have to specify `pending.dissolution` status. After some check would have been done, lot status will be switched to `dissolved` automatically.

File: docs/source/tutorial.rst [hunk 3]
```diff
@@ -138,6 +139,12 @@ attached to another lot), status of the current one will be turned to `pending`:
 .. literalinclude:: tutorial/concierge-patched-lot-to-pending.http
    :language: javascript
 
+When bot finds that status of a lot is `pending.dissolution`, it will 
+make status of lot assets to pending and change lot status to `dissolved`.
```
Comment (ivanka-jakimchuk) [inline docs/source/tutorial.rst]: ......., turn status of the assets being attached to that lot to `pending`. Status of the lot itself will become `dissolved`.

File: docs/source/tutorial.rst (hunk 2 above)
Comment (oleksiyVeretiuk) [reply docs/source/tutorial.rst]: Зроблено в https://github.com/openprocurement/openregistry.lots.basic/pull/39/commits/8f89d657c72ac78dc27427cb30ab351009ef5dff

File: docs/source/tutorial.rst (hunk 3 above)
Comment (oleksiyVeretiuk) [reply docs/source/tutorial.rst]: Зроблено в https://github.com/openprocurement/openregistry.lots.basic/pull/39/commits/8f89d657c72ac78dc27427cb30ab351009ef5dff

**Pull request merged**